api/envoy/config/filter/http/compressor/gzip/v2/gzip.proto

```diff
@@ -0,0 +1,62 @@
+syntax = "proto3";
```

Comment:
How does this relate to https://github.com/envoyproxy/envoy/blob/master/api/envoy/config/filter/http/gzip/v2/gzip.proto? Which one should service operators configure and when? Is the former deprecated?

Reply:
Yes, exactly, the former is deprecated. We need to make that clear as @htuch points out, have text in deprecated.rst, and also make sure the deprecated feature warning/stat is incremented so that operators know thy need to upgrade.